PowerPointLabs/PowerPointLabs/Utils/Graphics.cs

```diff
@@ -81,13 +81,14 @@ public static void ExportShape(ShapeRange shapeRange, string exportPath)
 
         public static Bitmap ShapeToImage(Shape shape)
         {
-            string fileName = "synclabtemp.png";
+            string fileName = TextCollection.SyncLabImageStorageFileName;

```

Comment:
This method resides in Graphics which may be used by various labs, so it might be better to no specify synclab in the name.